ssc/cmod_geothermal_costs.cpp

```diff
@@ -295,6 +312,146 @@ class cm_geothermal_costs : public compute_module
 	void exec() override
 	{
 		SGeothermal_Inputs geo_inputs;
+
+        int calc_drill_costs = as_integer("calc_drill_costs");
+        if (calc_drill_costs == 1) {
+            int inj_cost_curve_welldiam = as_integer("geotherm.cost.inj_cost_curve_welldiam");
```

Comment:
shouldn't these inputs be required if "calc_drill_costs"==1?

Reply:
Yes, I will update this.